dev/CommonStyles/Common_themeresources_any.xaml

```diff
@@ -570,16 +570,102 @@
             <SolidColorBrush x:Key="CircleElevationBorderBrush" Color="{ThemeResource SystemColorWindowTextColor}" />
             <SolidColorBrush x:Key="AccentControlElevationBorderBrush" Color="{ThemeResource SystemColorWindowTextColor}" />
 
-            <!-- System high contrast colors -->
             
-            <SolidColorBrush x:Key="SystemColorWindowTextColorBrush" Color="{ThemeResource SystemColorWindowTextColor}" />
-            <SolidColorBrush x:Key="SystemColorWindowColorBrush" Color="{ThemeResource SystemColorWindowColor}" />
-            <SolidColorBrush x:Key="SystemColorButtonFaceColorBrush" Color="{ThemeResource SystemColorButtonFaceColor}" />
-            <SolidColorBrush x:Key="SystemColorButtonTextColorBrush" Color="{ThemeResource SystemColorButtonTextColor}" />
```

Comment:
These are used, so we cannot delete them. But we use them only in HighContract dictionaries. To be safe, I'd recommend duplicating them as well, but with a comment saying that these are high contrast brushes only for use in high contrast dictionary.